bin/aws-efs-creator.ts

```diff
@@ -15,7 +15,14 @@ const app = new cdk.App();
 const appAspects = Aspects.of(app);
 
 // check environment variables
-checkEnvVariables('APP_NAME', 'OWNER', 'VPC_ID', 'CDK_DEPLOY_REGION', 'ENVIRONMENT');
+checkEnvVariables('APP_NAME',
+'OWNER',
+'VPC_ID',
+'CDK_DEPLOY_REGION',
+'ENVIRONMENT',
+'PERFORMANCE_MODE',
+'THROUGHPUT_MODE',
```

Comment:
**Suggestion:** Consider handling potential undefined values for the new environment variables `PERFORMANCE_MODE` and `THROUGHPUT_MODE` when calling `checkEnvVariables`. This will ensure that the application does not throw runtime errors if these environment variables are not set, especially since they are marked as potentially undefined in other parts of the code. [possible issue, importance: 9]
```suggestion
checkEnvVariables('APP_NAME',
'OWNER',
'VPC_ID',
'CDK_DEPLOY_REGION',
'ENVIRONMENT',
process.env.PERFORMANCE_MODE ? 'PERFORMANCE_MODE' : undefined,
process.env.THROUGHPUT_MODE ? 'THROUGHPUT_MODE' : undefined,
);
```